lib/api/WP_Auth0_Api_Change_Email.php

```diff
@@ -66,6 +66,7 @@ public function call( $user_id = null, $email = null ) {
 			->add_body( 'email', $email )
 			// Email is either changed by an admin or verified by WP.
 			->add_body( 'email_verified', true )
+			->add_body( 'client_id', $this->options->get( 'client_id' ) )
```

Comment:
[Recommend by the API docs for this endpoint](https://auth0.com/docs/api/management/v2#!/Users/patch_users_by_id)